src/styles/index.scss

```diff
@@ -1,3 +1,53 @@
-body {
-  margin: 0;
+@import './abstract/constants';
+@import './abstract/placeholders';
+@import './base/base';
+
+.calendar {
+  display: flex;
+  padding: $calendar-indent;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  @extend %calendar-width;
```

Comment:
It's not a good idea to use `@extend` with a placeholder that sets a width. It's better to set the width directly in `.calendar` class to make it more explicit and readable.